content/en/docs/tasks/observability/metrics/querying-metrics/index.md

```diff
@@ -40,7 +40,7 @@ the example application throughout this task.
     browser or issue the following command:
 
     {{< text bash >}}
-    $ curl http://$GATEWAY_URL/productpage
+    $ curl http://"$GATEWAY_URL"/productpage
```

Comment:
This works, but a nicer bash convention, IMO, is to put the quotes around the entire arg.

```suggestion
    $ curl "http://$GATEWAY_URL/productpage"
```